drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -62,6 +62,19 @@
 #define PLLSRC_FREQ 0
 #endif
 
+#define VCO_FREQ	(PLLSRC_FREQ / CONFIG_CLOCK_STM32_PLL_M_DIVISOR)
+#if (1000000UL <= VCO_FREQ && VCO_FREQ <= 2000000UL)
+#define VCO_INPUT_RANGE LL_RCC_PLLINPUTRANGE_1_2
+#elif (2000000UL < VCO_INPUT_RANGE && VCO_INPUT_RANGE <= 4000000UL)
```

Comment:
And noone noticed this mistake from copying :)

Reply:
Now it it more readable, I can see it ;-)

Reply:
Exactly that's a good one ! 🙃 I had a attentive look at the constants but didn't catch that one. I could have seen it by testing though ^^